parser.py

```diff
@@ -623,6 +632,13 @@ def main():
 
     users = {}
```

Comment:
`state["users"]` seems to duplicate `users`?

I like the idea of have a single cache mechanism for users, media, unshortened URLs, so let's delete `users` and use `state["users"]`. Unless I've missed something?

Reply:
It does not explicitly duplicate users but in effect it does.
 Good point removing that.